pkg/sources/client/azure/servicebus/client.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2023 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicebus
```

Comment:
I understand this will remain and the other copies will be removed.
Does not feel great to have copies around, let's remember to remove the one at servicebustopics

Alternatively we could remove it now and redirect the previous code to this client, but that is not needed now on my side.

Reply:
Yes, that's the plan, as soon as we delete the old azure servicebus topic and queue I'll remove this with the rest of the old code.